modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -151,6 +151,12 @@ public static void startup(Runnable runnable) {
      * {@link #startup(Runnable)} once.
      * </p>
      *
+     * <p>
+     * Memory consistency effects: Actions in a thread prior to submitting a
+     * runnable to this method <i>happen-before</i> actions performed by
```

Comment:
I noticed that other classes add the link to the concurrency package summary as in https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/util/concurrent/BlockingQueue.html

You might want to do so as well.

Reply:
I considered such a link but didn't add it. My reasoning is that for the standard library classes such a link is internal, to the same Javadoc site.

For JavaFX on the other hand such a link is external, pointing to a site outside the control of the JavaFX project. This has the effect, for example that the link must be kept up to date. Should it always point to the latest Java release? What if the URL change? What if the package summary changes format in a future version?

I think it is more sensible to leave out such a link. "_Happens-before_" is a standard concept in Java and readers can locate an explanation for it themselves.

Reply:
OK.